src/Modal/Modal.spec.js

```
@@ -1,26 +1,34 @@
 import React from 'react';
+import eventually from 'wix-eventually';
+
 import Modal from './Modal';
-import ModalFactory from './Modal.driver';
-import { createDriverFactory } from 'wix-ui-test-utils/driver-factory';
-import sinon from 'sinon';
-import {
-  isTestkitExists,
-  isEnzymeTestkitExists,
-} from '../../test/utils/testkit-sanity';
-import { modalTestkitFactory } from '../../testkit';
-import { modalTestkitFactory as enzymeMessageBoxTestkitFactory } from '../../testkit/enzyme';
-import { mount } from 'enzyme';
+import ModalFactory, {
+  isUnmountSideEffectsDone as isModalUnmountDone,
+} from './Modal.driver';
+
+import { ReactDOMTestContainer } from '../../test/dom-test-container';
+
+const MODAL_CLOSE_TIMEOUT = 10;
 
 describe('Modal', () => {
-  const createDriver = createDriverFactory(ModalFactory);
+  afterEach(async () => {
+    await eventually(() => expect(isModalUnmountDone()).toBeTruthy(), {
```

I think it's not recommended to make assertions in the `afterEach` hooks.

---

I can change it to throw an error. You need to throw an error in order for eventually to work.

---

I think it's enough to return a rejected promise

---

Ok, I will.